src/seedu/addressbook/Main.java

```diff
@@ -84,9 +84,13 @@ private void runCommandLoopUntilExitCommand() {
         do {
             String userCommandText = ui.getUserCommand();
             command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
-            recordResult(result);
-            ui.showResultToUser(result);
+            try {
+                CommandResult result = executeCommand(command);
```

Comment:
This commit doesn't belong to the 2KLoC exercise! Please branch properly going forward,